kclvm/sema/src/resolver/import.rs

```diff
@@ -211,4 +232,85 @@ impl<'ctx> Resolver<'ctx> {
         self.ctx.filename = filename.to_string();
         self.scope = self.scope_map.get(pkgpath).unwrap().clone();
     }
+
+    pub(crate) fn check_unused_import(&mut self, pkgpath: &str) {
+        match self.program.pkgs.get(pkgpath) {
+            Some(modules) => {
+                for module in modules {
+                    if let Some(import_names) = self.ctx.import_names.get(&module.filename) {
+                        let mut walker = UnusedImportCheckWalker {
+                            used_import: IndexSet::default(),
+                            import_names: import_names.clone(),
+                        };
+                        let mut m = module.clone();
```

Comment:
Do not use `clone()`, please considering `UnusedImportCheckWalker` use the `&module` instead of `&mut module`. A further problem is that the resolver itself is an `impl Walker`, which calls an `impl Walker`, they should be in a parallel relationship instead of a nested relationship.

Reply:
Constrcut a new walker for Lint, has removed this